tests/run.go

```diff
@@ -21,16 +21,24 @@ import (
 	"github.com/runfinch/common-tests/option"
 )
 
+// RunOption is the custom option to run tests in run.go.
+type RunOption struct {
+	BaseOpt    *option.Option
+	JourctlOpt *option.Option
```

Comment:
- so that folks don't have to dig into our code to understand the implications of not setting this field
- JourctlOpt  -> JournalctlOpt, probably typo

```suggestion
	// JournalctlOpt instructs how to run the journalctl binary. If it's nil, test cases related to `--log-driver journald` will be skipped.
	JournalctlOpt *option.Option
```